### PR TITLE
styling tweaks to the facets

### PR DESCRIPTION
**Pull request by aliceriot**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [x] Tag @ferdi or @pdpinch for review
- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?

part of #2161 

#### What's this PR do?

this restyles the facets to match the mockups. basically:

- new text styles
- clicking the title opens and closes the facet list

...that's about it, haha. I left off of this PR the restyling of the currently selected facets, just to keep the PRs smaller.

#### How should this be manually tested?

go to the course search page and make sure that the facets that you see are like they are on the mockups.

#### Screenshots (if appropriate)

facets2
facets

### CODE REVIEW

**Comment by aliceriot** [comment]
made a few styling tweaks:

coursearchtweaks

File: static/js/components/SearchFacet.js [hunk 1]
```diff
@@ -20,102 +15,83 @@ type OwnProps = {|
...
-      onUpdate,
-      showAll,
-      displayCount
-    } = this.props
-    const maxCount = displayCount || 5
+  return (
+    <div className="facets">
+      <div
+        className="facet-title"
+        onClick={() => setShowFacetList(!showFacetList)}
```
Comment (mbertrand) [inline static/js/components/SearchFacet.js]: Maybe add a unit test for this, otherwise looks/works great.

Comment (aliceriot) [reply static/js/components/SearchFacet.js]: ah, meant to do that but forgot! will add one.

**Comment by aliceriot** [comment]
@mbertrand ok I added that test real quick

**Pull request merged**